changelog/fragments/reorder-docker-args.yaml

```
@@ -0,0 +1,31 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Support folks that may have `docker` aliased to `podman`, and Podman prefers options before positional arguments:
+
+      ```console
+      $ podman build --help | grep CONTEXT-DIRECTORY
+         podman build [command options] CONTEXT-DIRECTORY | URL
+      ```
```

This can be simplified since the migration guide is thorough
```suggestion
      (ansible/v1, helm/v1) Post-fixed positional directory argument `.` in `docker-build` make target to align with `podman`
```